.github/stale.yml

```diff
@@ -0,0 +1,17 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - pinned
+  - security
```

Comment:
how 'bout adding triaged here?

Reply:
Currently, most of the issues are marked as triaged so the bot wil do nothing in that case.

Reply:
We'd need to redefine how we use it because currently it means "someone has taken a look at it" - so we add it to basically any issue.

Reply:
So. What about removing `triaged` from all issues?